dascore/core/coords.py

```
@@ -1240,9 +1240,16 @@ def update(self, **kwargs):
         """No values to change so update can just call new."""
         return self.new(**kwargs)
 
-    # Other operations that normally modify data do not in this case.
-    update_limits = update
-    set_units = update
+    # Other operations that normally modify data do not in this case;
+    # they are spelled out rather than aliased so each keeps the
+    # signature its base declares.
+    def update_limits(self, min=None, max=None, step=None, **kwargs) -> BaseCoord:
```

Ruff (0.16.1)

[error] Function argument `min` is shadowing a Python builtin

(A002)